linera-chain/src/chain.rs

```diff
@@ -566,8 +565,10 @@ where
                 BlockHeight(0),
                 local_time,
             )?;
+            Ok(true)
+        } else {
+            Ok(false)
```

Comment:
```rust
let Message::System(SystemMessage::OpenChain { … }) = message else {
    return Ok(false);
};
```

might be easier to read. (Then you don't have to remember at the end what the `else` refers to.)

Reply:
Sadly, `rustfmt` wants to break before `else` but ok.